### DISCUSSION TITLE
This repository is temporarily unavailable

### DISCUSSION THREAD

**Discussion by nlg550** [community/community#61348] [Repositories]
### Select Topic Area

Bug

### Body

Hello,

I changed [my repository](https://github.com/Aesfrost/MOCCA) from public to private. But I got stuck on "This repository is temporarily unavailable. This repository is moving to a newer, better home. It’ll be back in just a few minutes.". How I solve this issue?

![image](https://github.com/community/community/assets/38725499/8d307f83-740e-426e-bd84-245c3915567f)

**Comment by ClassyJuhi** [comment]
1. Wait: Give it some time as the repository is being moved, which might take a few minutes.

2. Clear Cache: Clear your browser cache to ensure you're not viewing a cached version of the repository.

3. Check Internet Connection: Ensure you have a stable internet connection to access the repository.

4. Logout and Login: Log out of your account and log back in to refresh your session.

5. Verify Settings: Double-check that you changed the repository's visibility from public to private in the settings.

**Reply by choi-junhee** [reply]
It didn't work for me. It seems like a problem with the server because I waited for about 2-3 weeks.

**Reply by unmaykr-aftermath** [reply]
Same thing for us, been **hours** since we made https://github.com/AftermathFinance/aftermath-sdk-rust public but it's stuck with the same message

**Reply by unmaykr-aftermath** [reply]
I opened a ticket with GitHub support. Hoping they reply to us soon

**Reply by unmaykr-aftermath** [reply]
Opening a ticked worked! They got back to me in an hour or so and 'unlocked' the repo

> Thanks for reaching out to GitHub Support.
>
> I have unlocked the repository which was locked due to migration/change of visibility.
